### DISCUSSION TITLE
Ошибка компиляции кода на esp8266

### DISCUSSION THREAD

**Discussion by Konstantin8920** [vvip-68/GyverPanelWiFi#292] [Задавайте вопросы]
подскажите пожалуйста как решить вот этот вопрос на плате на ESP266MOD на платформе nodemcu V3 на чипе CH340C, при компиляции выдает ошибку 

fork/exec C:\Users\kos89\AppData\Local\Arduino15\packages\esp8266\tools\xtensa-lx106-elf-gcc\2.5.0-4-b40a506/bin/xtensa-lx106-elf-g++.exe: Access is denied.

Compilation error: fork/exec C:\Users\kos89\AppData\Local\Arduino15\packages\esp8266\tools\xtensa-lx106-elf-gcc\2.5.0-4-b40a506/bin/xtensa-lx106-elf-g++.exe: Access is denied.  
версия ядра 2.7.4
пробовал удалять драйвера на плату и устанавливать обратно, 
пробовал обновлять до последней версии,
пробовал более старые версии устанавливать, 
ничего не помогло, одна надежна на вас)) 
заранее огромное спасибо!!

**Comment by vvip-68** [comment]
Тут ни от чипа ни от драйверов ни от версий вообще не зависит.
Что говорит компилятор/линкер - что один из его файлов заблокирован и ему не удается его запустить. 
Тут или файла реально нет - тогда нужно переустановить ядро кмикроконтроллера - удалить его из Arduino ID и поставить заново, или смотреть кто блокирует доступ к файлу - какой-нибудь антивирус или что-то в этом роде. Тогда внести в исключения или отключить антивирус на время...

**Reply by Konstantin8920** [reply]
Спасибо за совет, решил проблему таким образом, Снес все что касается Arduino ID (стояла последняя версия) потом установил версию 1.8.13 Arduino ID  и скетч скомпилировался )))
